### ISSUE TITLE
Tag gets automatically generated from links containing # character

### ISSUE THREAD

**Issue by sfr0xyz** [verbiricha/habla.news#163]
If you have an link to an website and the URL contains a hashtag #, for instance linking to an Wikipedia subheading like https://en.wikipedia.org/wiki/Nostr#Open_protocol, Habla automatically generates a tag, in the example case `#Open_protocol`.

**Comment by verbiricha** [comment]
Good catch, seems to be an issue with the [NDK hashtag regex](https://github.com/nostr-dev-kit/ndk/blob/6d92481d8974dda381cf47ddf79a155e3dde663b/ndk/src/events/content-tagger.ts#L79)
